lib/graph3d/Graph3d.js

```diff
@@ -62,14 +63,16 @@ Graph3d.DEFAULTS = {
   animationPreload     : false,
   animationAutoStart   : autoByDefault,
 
+  axisFontSize     : 14,
+  axisFontType     : 'arial',
   axisColor        : '#4D4D4D',
   gridColor        : '#D3D3D3',
   xCenter          : '55%',
   yCenter          : '50%',
 
   style            : Graph3d.STYLE.DOT,
   tooltip          : false,
-
+  tooltipDelay     : 150,
```

Comment:
What is a reason to change `tooltipDelay` from 300 (https://github.com/almende/vis/blob/5563fc616885578ad51b2dfa6ccf70ccebf06126/lib/graph3d/Graph3d.js#L2003) to 150 in this PR ?
It seems not related to rotating axis, so I thinks the change should be splitted into separate PR.

Reply:
Sure thing! I had lowered it for testing to go quicker when changing values.